slack-base/src/main/java/com/hubspot/slack/client/models/blocks/InputIF.java

```diff
@@ -34,4 +34,6 @@ default String getType() {
 
   @JsonProperty("optional")
   Optional<Boolean> isOptional();
+
```

Comment:
Do we need @JsonProperty("dispatch_action") ?

Reply:
No, having `@JsonNaming(SnakeCaseStrategy.class)` annotation is enough.

Reply:
Checked if it works and it works!